1_fetch.R

```diff
@@ -96,12 +96,12 @@ p1_targets_list <- list(
     format = "file"
   ),
   
-  # Pull site id's from the WQP inventory
+  # Pull site id's and total number of records for each site from the WQP inventory
   tar_target(
     p1_site_ids,
```

Comment:
We may want to also rename this to make it clear that it is no longer just a vector of sites. Maybe `p1_site_counts` or something?

Reply:
Yeah, I like that idea, and your suggestion of `p1_site_counts`. I've also changed the name of the downstream target from `p1_site_ids_grouped` to `p1_site_counts_grouped`.